benchmarks/triton_kernels_benchmark/gemm_postop_addmatrix_benchmark.py

```diff
@@ -247,29 +262,42 @@ def matmul(a, b, d, c):
         # name for the plot. Used also as a file name for saving the plot.
         args={},
     ))
-def benchmark(B, M, N, K, provider):
+def benchmark(B, M, N, K, dtype, provider):
+    res_dtype = torch.float32 if dtype is torch.bfloat16 else torch.int32
```

Comment:
to be consistent with the code above?
```suggestion
    res_dtype = torch.float32 if a.dtype.is_floating_point else torch.int32
```

Reply:
I am not sure if `float32` would be the correct res_dtype for all floating point so I think we should leave it explicit so anyone adding additional dtypes knows to change the result dtype accordingly.

Reply:
Then should we change references of `tl.float32 if a.dtype.is_floating_point else tl.int32` with `tl.float32 if a.dtype is tl.bfloat16 else tl.int32`?

Reply:
I don't understand why it is preferable to access `a.dtype` instead of looking at the `dtype` parameter that is passed in?

Reply:
I see - I'm fine with `dtype.is_floating_point` if you prefer that over `dtype is torch.bfloat16`. WDYT?

Reply:
I prefer consistency, so changing line 211 and 226 to `dtype is torch.bfloat16`, or line 275 to `dtype.is_floating_point` both works for me.

Reply:
Done